package.json

```diff
@@ -72,7 +72,7 @@
   "scripts": {
     "=============================================================================== code hygene": "",
     "lint": "check() { eslint ${1:-ts/}; }; check",
-    "lint:fix": "fix() { eslint --fix ${1:-ts/}; -- format unix}; fix",
```

Comment:
I think it may have been intended as `--format unix` (no space after `--`), but I'm OK with removing it.

Reply:
I've pivoted twice on this one. Regardless of the space, the order would have been incorrect anyway. But we really only need the unix formatting, when in Emacs. On the console the regular one looks better.